### DISCUSSION TITLE
No lock-in

### DISCUSSION THREAD

**Discussion by MD77MD** [immich-app/immich#14215] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Stores the metadata (description, tages, etc) in the EXIF headers of photos, which means that we can easily migrate to other solutions without losing our data.

this is a very important feature for any open-source app.

The "No lock-in" feature gives me a peace of mind that all my efforts of organizing, tagging, Describing... etc of my assets is always thier and will never be lost because everything is stored in the EXIF metadata.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Most metadata edits are already stored in xmp sidecar files, and the goal is to do that for all metadata where possible.

**Comment by MD77MD** [comment]
are tags included

**Reply by alextran1502** [reply]
Yes

**Reply by MD77MD** [reply]
then i think my problem is with the import of EXIF metadata when uploading assets as it doesn't import all the metadata like tags, titles or descriptions 

and because of that i though its a problem with immich as a whole
